examples/using-emotion-prismjs/src/utils/typography.js

```diff
@@ -0,0 +1,12 @@
+import Typography from "typography"
+import bootstrapTheme from "typography-theme-bootstrap"
+console.log(bootstrapTheme)
+
+bootstrapTheme.googleFonts = [{
+    name: `Source Sans Pro`,
```

Comment:
Did you mean `Source Code Pro`?

Reply:
I did. This is fixed now. Thanks for catching that!